core/arch/arm/plat-imx/conf.mk

```diff
@@ -3,27 +3,23 @@ include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 CROSS_COMPILE	?= arm-linux-gnueabihf-
 COMPILER	?= gcc
 
-CFG_ARM32_core ?= y
-CFG_MMU_V7_TTB ?= y
-
-core-platform-cppflags	 = -I$(arch-dir)/include
+core-platform-cppflags  = -I$(arch-dir)/include
+core-platform-cppflags += -D_USE_SLAPORT_LIB
+core-platform-cppflags += -DCFG_NO_TA_HASH_SIGN
```

Comment:
`_USE_SLAPORT_LIB` and `CFG_NO_TA_HASH_SIGN` can be removed. May be in another pull-request.